docs/builddocs.sh

```diff
@@ -49,7 +49,7 @@ build_api_docs() {
   # Then build for devsite
   $DOCFX build --logLevel Warning --disableGitFeatures output/$api/devsite-docfx.json | tee errors.txt | grep -v "Invalid file link"
   (! grep --quiet 'Build failed.' errors.txt)
-  mv output/$api/devsite/toc.html output/$api/devsite/_toc.yaml
+  mv output/$api/devsite/api/toc.html output/$api/devsite/api/_toc.yaml
```

Comment:
How difficult would it be to change the output directory to include the API name referenced in `_rootPath`?

Reply:
The output directory is irrelevant here, given that we're going *into* that directory to perform the upload. If you want to be "above" the output directory when we do the upload, I think we'll need to do more work - we'll want to plumb it through a little more carefully.